README.md

```diff
@@ -43,35 +43,86 @@ It will create some environment variables to ease access to some folders:
 ```bash
 source WRemnants/setup.sh
 ```
+### Theory agnostic analysis
 
+Make histograms (only nominal and mass variations for now, systematics are being developed)
+```
+/usr/bin/time -v python scripts/histmakers/mw_with_mu_eta_pt.py -o outputFolder/ --met DeepMETReso  --addHelicityHistos --pt 30 26.0 56.0 --onlyMainHistograms
+```
+
+Prepare datacards and root files with TH2 (stat-only for now)
+```
+/usr/bin/time -v python scripts/combine/setupCombineWMass.py -i outputFolder/mw_with_mu_eta_pt_scetlib_dyturboCorr.hdf5  -o outputFolder/  --absolutePathInCard --theoryAgnostic --genAxis absYVgenSig ptVgenSig helicity --doStatOnly
```

Comment:
You might want to auto set the genAxes that are needed when --theoryAgnostic is used. I would in general try to make the command line args as simple as possible, so if only --absolutePathInCard and --doStatOnly are supported, auto set them and then print a logger message about it

Reply:
I personally believe that  --absolutePathInCard is more convenient for any user that uses their own stuff, but having relative path was deemed the best when cards are shared within multiple people and thus it eventually became the new default, so I simply added a new option to reproduce my previous preferred behaviour.
You are right that --doStatOnly is currently the only way to run the code, but this will hopefully change soon, so I would not make it automatically set inside (in fact we can already use some systematics, so in principle this option would not be always needed). Again, the purpose here was to share with some people the commands I used so far, so we could all get consistent outputs.

I keep note that these explicitly setting these options won't be needed in the near future.

Reply:
So --absolutePathInCard is fine. But I would still make it force --doStatOnly for now and change it later. I maintain that the simpler the instructions the better. Ideally running the code with no args gives the correct result